### ISSUE TITLE
Veteran transition platform/app 

### ISSUE THREAD

**Issue by christyleos** [TechForJustice/project-ideas#27]
Create a interactive tool for veterans who are transitioning or already have transitioned out of the military. This tool should help users plan: 
- finances
- home/residence
- education
- training
- employment
- benefits
- mental health upkeep

Possible road maps to follow: 
- The [Veterans Transition Map & Action Plan](http://purplestarveterans.org/transition-tools-resources/) by Purple Star
- [Prepare for Transition](http://www.wtc.army.mil/modules/veterans/v11-prepareTransition.html) by Warrior Care and Transition
- I[ndividual Transition Planning](http://www.military.com/military-transition/7-phases-of-idp.html) by Military.com 

**By helping vets with these areas, we could prevent hardships that can lead to more debilitating problems, such as homelessness, suicide, and violence.** 

The VA's [TAP ](http://www.benefits.va.gov/vow/tap.asp)program ( Transition Goals, Plans, Success program—known as Transition GPS) is the current solution and was developed by the Department of Defense in coordination with the Department of Veteran Affairs. The program provides comprehensive services to service members to transition to work, life, and home after the military. However, after speaking to the focus group who have developed the problem set for the Veteran's hackathon, it isn't incredibly helpful. 

@jakierice @JeffL-T your thoughts?

**Comment by JeffL-T** [comment]
The challenge is too bring together a broad scope of complex transition resources together in an easy to understand interface that can be intuitively navigated and actioned.  A fully comprehensive transition online assistance program should continue to serve the veteran beyond the date of separation.  As the Purple Star organization referenced above points out, "Veteran Peer Advisors" are needed to help direct veterans to the most relevant resources.

I can envision combining a well-organized online directory with an A/V interface to the veteran's peer advisor, a text message interface to the advisor, file sharing, and a customizable resource directory.

**Comment by christyleos** [comment]
There is a current solution being made: http://victorapp.io/

**Comment by JeffL-T** [comment]
It is still at concept stage, is that right?  We need to run it through is
paces and see what it can and can't do.

Could Bouy be layered in somehow?  An alert to your friends, family, or
treatment team when in crisis?  Just a thought.  (Bouy's anti-police &
anarchist leanings make me a bit nervous though, and I say that as an Abbey
Hoffman fan.  I think a world without police would be great as long as we
all behaved).

**Comment by christyleos** [comment]
Yes still conceptual but they had a great fundraising campaign, so we can assume it will be built.

**Comment by JeffL-T** [comment]
*moved over from issue #28

MigrantCloth: Hi.

Talk about perfect timing! I've been putting together an ebook to help vets better prefer for/manage TAPS (I'm a USMC vet). The intent was to use that and a landing page and funnel the audience to a website built to create a platform to aggregate resources I am a tech noob as I just started learning last year but (1) I'm a project coordinator and business analyst and (2) I intend to further build my tech skills to help veterans. 

Webframe idea:
[EAS Countdown.pptx](https://github.com/TechForJustice/project-ideas/files/802408/EAS.Countdown.pptx)

Intent: Veteran can enter their EAS date, the clock counts down. The buttons below the countdown will redirect to resources such as organizations for claims (VFW, AMVETS,etc), education steps, finance steps, and so on. 

When I first looked to address this issue, I saw individual issues I've helped veterans with or issues I had and there really is a broad range. I settled on starting on guidance to better manage TAPS(including teaching Cornell note-taking and thorough career search using an MOS engine and a job engine) as a way to both keep my scope small and get feedback. Then I I figured I could build 'backward' to creating a platform for veterans to better manage their entire careers. I think too broad a scope up front not only 'bites off more than you can chew' but also would give the audience too much at once to take in (and most already feel that way about the TAPS experience). 

What are you thoughts?

**JeffL-T:** Hello MigrantCloth,

An EAS Countdown is a great idea to help relieve the volume of tasks facing members at separation. Project idea 27, Veteran transition platform/app, is listed prior to this Chatbot idea and would be a good place to discuss your countdown idea. Do you have any objection to me moving it there? Also there, you will see a link to a transition tool being developed by Bunker Labs called VictorApp. It looks like a good app, but I can't see that it has a countdown feature like you have conceived. Such a feature would have tremendously aided my own transition so I think that it is a superb idea.

**MigrantCloth:** Hey Jeff,

No, that's fine. Sounds great!

I did get a chance to check out the website for VictorApp. I'm not able to see the app (Android) but from everything on the site, it looks good. Even if we're different sides of the same coin (or different sections of the same side) I think the countdown and interactive goal-setting journey through resources idea would be a great asset to any transition platform. This is what I'm hoping to achieve with EAS Countdown.

**Comment by JeffL-T** [comment]
The BIG IDEA is to create a virtual post-separation assistant to better present the vast array of detailed and complex transition resources together in an easy to understand way that can be intuitively navigated and actioned.

**Comment by MigrantCloth** [comment]
Jeff/All,

Is this project active today?

**Comment by JeffL-T** [comment]
Hi MigrantCloth,

I have not seen any activity on this project today.

**Comment by christyleos** [comment]
@MigrantCloth have you asked around on the slack? https://techforjustice-community.herokuapp.com/
